bioio_ome_zarr/writers/config.py

```diff
@@ -0,0 +1,106 @@
+from typing import Any, Dict, Union
+
+import dask.array as da
+import numpy as np
+
+from .utils import chunk_size_from_memory_target
+
+
+def get_ome_zarr_writer_config_for_viz(
```

Comment:
I like how specific this is - but it is rather long - maybe `get_default_config_for_viz` and `get_default_config_for_ml`